src/lug/Window/Unix/WindowImplX11.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <lug/Window/Window.hpp>
+
+namespace lug {
+namespace Window {
+
+namespace priv {
+class WindowImpl {
```

Comment:
Where are the move constructor and move assignment operator? (`= delete` I think)